### ISSUE TITLE
Modules configuration strategy

### ISSUE THREAD

**Issue by ironsmile** [ironsmile/nedomi#26]
We have to have a clear and single module configuration strategy. The way the logger module does it at the moment does seem like the best one. I just want to document the strategy here, in the wiki or in the README.

**Comment by na--** [comment]
This is mostly done, right?

**Comment by ironsmile** [comment]
Yes, I think so. But it is not written anywhere so that future module writers (such as myself) know what to do.
